src/modules/data/blocks/ticket/sagas.js

```diff
@@ -285,20 +288,58 @@ export function* fetchTicketDetails( action ) {
 	const { ticketId = 0, blockId } = action.payload;
 
 	yield put( actions.setTicketIsLoading( blockId, true ) );
-
 	try {
+		// Prevent to fetch details of a non created ticket
 		if ( ticketId === 0 ) {
+			yield call( removeBlock,  blockId );
 			return;
 		}
 
-		const ticket = yield call( wpREST, {
+		const { data = {}, response = {} } = yield call( wpREST, {
 			path: `tickets/${ ticketId }`,
 			namespace: 'tribe/tickets/v1',
 		} );
+		const ticket = data;
+
+		if ( response.status === 404 ) {
+			yield call( removeBlock, blockId );
+			return;
+		}
+
 
 		const costDetails = ticket.cost_details || {};
 		const costValues = costDetails.values || [];
-		const { totals = {} } = ticket;
+		const {
+			totals = {},
+			available_from,
+			available_from_start_time,
+			available_until,
+			available_from_end_time,
```

Comment:
small thing, but are these names something we have control over (defined in the php somewhere)? If so, is it possible to make them consistent (for example, `available_from`, `available_from_time`, `available_until`, and `available_until_time` (or `available_to` and `available_to_time`))? If it's a part of ET and is a huge lift don't worry about it.

Reply:
Unfortunately is something that has been already release and is part of a tickets API that is part of tickets plugins already so it's nothing new created on Gutenberg.